main/ahb_for_sles.go

```diff
@@ -421,13 +468,17 @@ func getExtensionAndRun() error {
 	initilizationInfo.UpdateCallback = updateCallbackFunc
 	vmExt, err := getVMExtensionFuncToCall(initilizationInfo)
 	if err != nil {
+		fmt.Fprintln(os.Stderr, "Extension initialization failed. Reason="+err.Error())
+		vmExt.ExtensionEvents.LogErrorEvent(
+			"Initialization",
```

Comment:
I understand `Initialization` is not an operation available/listed in the HandlerManifest but maybe we could add 
`INITIALIZE_EVENT:     "Initialization"` to the [consts in line 49](https://github.com/SUSE-Enceladus/ahb-extension/pull/15/files#diff-4fd45add4912cb8303f57a306b692c86cdb8e0ec31ad3d564b867f725eb84161R49) ?

Reply:
Pushed the change

Reply:
Thank you for the changes